src/commitlint/cli.py

```diff
@@ -119,50 +126,59 @@ def _get_commit_message_from_file(filepath: str) -> str:
         return commit_message
 
 
-def _handle_commit_message(commit_message: str, skip_detail: bool) -> None:
+def _handle_commit_message(
+    commit_message: str, skip_detail: bool, quiet: bool = False
+) -> None:
     """
     Handles a single commit message, checks its validity, and prints the result.
 
     Args:
         commit_message (str): The commit message to be handled.
         skip_detail (bool): Whether to skip the detailed error linting.
+        quiet (bool): Whether to print the detailed errors
 
     Raises:
         SystemExit: If the commit message is invalid.
     """
     success, errors = lint_commit_message(commit_message, skip_detail=skip_detail)
 
-    if success:
-        sys.stdout.write(f"{VALIDATION_SUCCESSFUL}\n")
-    else:
-        _show_errors(commit_message, errors, skip_detail=skip_detail)
+    if quiet and errors:
         sys.exit(1)
 
+    elif not quiet:
```

Comment:
nitpick: Since the above if statement exits, we can use the `if` statement here.